app/src/main/java/com/keylesspalace/tusky/entity/Status.kt

```diff
@@ -149,6 +149,71 @@ data class Status(
         return builder.toString()
     }
 
+    /**
+     * overriding equals & hashcode because Spanned does not always ompare correctly otherwise
```

Comment:
typo in "compare"

Reply:
Same thing about `content` as before, I think it should either be a concrete impl of `Spanned` or just plain text.